corehq/apps/hqadmin/models.py

```diff
@@ -21,6 +21,9 @@ class HqDeploy(models.Model):
     environment = models.CharField(max_length=100, null=True)
     diff_url = models.CharField(max_length=255, null=True)
 
+    class Meta():
+        ordering = ["date"]
```

Comment:
It might be nicer for this to be `-date`, you generally want to view the last X deploys, not the first X deploys

Reply:
although that is quite similar to the method below

Reply:
Oh wow...the method below is actually what i usually want and I didn't notice it while writing this. But I don't want to have to remember a method name. Added 81f331ee9ab